### PR TITLE
Synced Version of Codebase

### PR DESCRIPTION
**Pull request by kimchiuu**
## **Types of Changes**
- [x] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Refactor (change which changes the codebase without affecting its external behavior)
- [x] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## **Purpose**
- Implement log out button feature, input and showing caloric goal of user
- Give some styling to our web application
- Fix merge conflicts and bugs in codebase

## **Approach**
- Created Redirect routing and changing state to successfully log in
- Integrated post request to backend to store the caloric goal of the user and render it to the frontend
- Utilized scss for styling

## **How to Test**
- npm install
- npm run dev
- on the Profile page, check if inputting the caloric goal works (look at console)

## **Screenshot(s)**
**Profile Page with Caloric Goal**
Screen Shot 2021-04-01 at 12 30 16 AM

**Feed Page with styling**
Screen Shot 2021-04-01 at 12 30 57 AM

Collaborated and Co-Authored:
@khuongdn16
@koderka2020
@tedtaemin

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,3 @@ node_modules/
 build/
 package-lock.json
 config.js
```
Comment (khuongdn16) [inline .gitignore:4]: @kimchiuu Can you add .env and make sure the commit does not have .env file

**Review by khuongdn16 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (kimchiuu) [reply .gitignore]: added .env to .gitignore

**Review by khuongdn16 — APPROVED** [review]
Approved

**Comment by tedtaemin** [comment]
LGTM

**Pull request merged**